docs/source/methods/avas.rst

```diff
@@ -168,6 +168,241 @@ We then use this guess of active orbitals to compute the CASCI energy:::
 For more examples, see :code:`avas-1` to :code:`avas-6` in the :code:`tests/methods` folder.
 In particular, :code:`avas-6` is a practical example on ferrocene.
 
+Defining the molecular plane for π orbitals
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Molecular systems with conjugated π bonds generally process planar geometries.
+For such systems, the active orbitals are usually chosen as the π orbitals perpendicular to the plane.
+Each π orbital consists of atomic p orbitals, which are also perpendicular to the plane.
+Take the above formaldehyde as an example.
+The π and π* orbitals are built from the 2p orbitals of C and O that are perpendicular to the molecular plane.
+We are very lucky that the formaldehyde molecule is in yz plane and
+the subspace 2p orbitals can be easily chosen as the 2px orbitals of C and O atoms.
+
+In reality, such perfect alignment does not happen often.
+Consider the below formaldehyde geometry ::
+
+  molecule H2CO{
+  C        0.346146295209737    0.126698337466632   -0.472844632676369
+  O       -0.346053592352928   -0.126664405871036    0.472717998223964
+  H        1.227335215970831   -0.489581944167415   -0.737753271803415
+  H        0.143281782803090    0.991262584630455   -1.134544367433545
+  noreorient  # ask Psi4 not to reorient the xyz
+  }
+
+Now the plane normal points to (1.0, 1.0, 1.0).
+Unfortunately Psi4 does not recognize the symmetry anymore unless we let it reorient the coordinate.
+However, you should confirm that the bond lengths and angles are identical to those in the previous geometry.
+If we use the same input for AVAS as above, here is what we get ::
+
+  ==> Atomic Valence MOs (Active Marked by *) <==
+
+    ===============================
+     Irrep    MO  Occ.  <phi|P|phi>
+    -------------------------------
+    *   A      0    2      0.905072
+    *   A      1    2      0.621379
+    *   A      8    0      0.374453
+    *   A      9    0      0.087267
+    ===============================
+
+Pay attention to the overlap metric.
+The projector becomes less effective and it is unclear which is the π orbital.
+The π* orbital also has a small overlap (0.374), much smaller than that from the previous aligned geometry (0.993).
+The reason for this is simply the subspace orbitals (i.e., 2px of C and O)
+are not perpendicular to the molecular plane.
```

Comment:
I think this example does not necessarily help. Sure, it explains what happens when you do the wrong thing and invoke AVAS on a nonplanar geometry. But this is something that I would explain pro-actively, in the sense that if we want the user to avoid an error, we should provide a list of few important things to do before running AVAS. I would jump right directly to the next section and directly say "The approach described in the previous section is not flexible enough to treat general π systems like molecules containing multiple π systems or π systems that are not aligned with a specific molecular axis."